enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -521,7 +546,11 @@ async def _launch_kernel(self, kernel_cmd, **kwargs):
                 self.kernel_spec.display_name, kernel_cmd
             )
         )
-
+        self.log.debug(
+            "Launching kernel: {} with command: {}".format(
+                self.kernel_spec.display_name, kernel_cmd
+            )
+        )
```

Comment:
This is a duplicate of the lines immediately before it.

Reply:
yes..ll remove